zabbix_metrics_watcher.py

```diff
@@ -214,14 +217,33 @@ def run(zabbix_host, osbs_master, config, instance):
 
             logger.info("Found build %s in %s", build_name, status)
             build = Build(build_name, cmd_base)
-            if status == 'Pending':
+            if status == 'New':
+                now = datetime.datetime.now()
+                if build_name not in builds_in_new:
+                    builds_in_new[build_name] = now
+
+            elif status != 'New':
+                try:
+                    del builds_in_new[build_name]
```

Comment:
Yes. However, the difference in behavior is that it would send the `0` value on every loop iteration when there are not builds in `New`, as opposed to the current behavior of sending `0` value only when the last build in `New` moves along.